api-gateway/http.go

```diff
@@ -68,12 +79,9 @@ func Router() http.Handler {
 	// Send requests to LLM for fraud detection, then kick off transaction
 	router.POST("/api/v1/transactions", createTransactionHandler)
 
-	// Handling assets
-	router.GET("/", assetHandler)
-	router.GET("/config.json", assetHandler)
-	router.GET("/favicon.ico", assetHandler)
-	router.GET("/assets/*asset", assetHandler)
-	router.GET("/images/*image", assetHandler)
+	// Anything that isn't a registered route should be served as a static file
+	router.NotFound = getStaticAssetHandler(assets)
```

Comment:
This is apparently the way to do chaining with [julienschmidt/httprouter]. There is a note in there about serving static files, but that assumes that you are not using an SPA that will handle routing (and 404s) on the client side.

[julienschmidt/httprouter]: https://github.com/julienschmidt/httprouter/tree/ef8578da8d8e1217ea44313b23edf46154b198ab#chaining-with-the-notfound-handler